src/bp/ui-studio/src/web/components/ContentForm/index.tsx

```diff
@@ -45,9 +47,34 @@ const widgets = {
   BaseInput: CustomBaseInput
 }
 
+// TODO: Remove this once audio and video content-types are support on multiple channels
+const CustomDescriptionField = ({ description, id, formContext }) => {
+  if (id === 'root__description' && ['audio', 'video'].includes(formContext.subtype)) {
+    const capitalize = (str: string) => {
+      return str.charAt(0).toUpperCase() + str.slice(1)
+    }
+
+    return (
+      <div id={id} style={{ lineHeight: 'normal' }}>
+        <div>
+          <span style={{ color: Colors.ORANGE3 }}>
```

Comment:
Can you use our color for this instead of using Blueprint color. I think the css color is 

`var(--lighthouse)`